Use the “python2” command on Arch Linux because “python” refers to version 3,
and this package isn’t compatible with that version.

--- PenguinTV.in	2011-11-21 06:55:15.000000000 +0000
+++ PenguinTV.in	2011-11-21 06:56:06.000000000 +0000
@@ -26,7 +26,7 @@ fi
 HILDON=0
 RUNMOZ_DIR=""
 if [ "$PYTHON"x = "x" ] ; then
-	PYTHON="/usr/bin/env python"
+	PYTHON="/usr/bin/env python2"
 fi
 
 if [ -f "/usr/lib/libgtkembedmoz.so.0" -a -d "/usr/lib/microb-engine" ]; then
